demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -15,64 +15,64 @@ class ReadMeValidator:
+        self.content_path = get_content_path()
+        self.file_path = Path(file_path)
+        self.pack_path = self.file_path.parent
 
-    def is_valid_file(self):
+    def is_valid_file(self) -> bool:
         """Check whether the readme file is valid or not
+
+        Returns:
+            bool: True if env configured else Fale.
         """
-        if os.environ.get('DEMISTO_README_VALIDATION'):
-            is_readme_valid = all([
-                self.is_mdx_file(),
-            ])
-            return is_readme_valid
+        if os.environ.get('DEMISTO_README_VALIDATION') or os.environ.get('CI'):
+            return self.is_mdx_file()
         else:
             return True
 
     def is_mdx_file(self) -> bool:
-        ready, is_valid = self.are_modules_installed_for_verify()
-        if ready:
-            mdx_parse = f'{os.path.dirname(os.path.abspath(__file__))}/../mdx-parse.js'
+        valid = self.are_modules_installed_for_verify()
+        if valid:
+            mdx_parse = Path(__file__).parent.parent / 'mdx-parse.js'
             # run the java script mdx parse validator
-            res = subprocess.run(['node', mdx_parse, '-f', self.file_path], text=True, timeout=10,
-                                 capture_output=True)
-            if res.returncode != 0:
-                print_error(f'Failed verifying README.md, Path: {self.file_path}. Error Message is: {res.stderr}')
-                is_valid = False
-        return is_valid
+            _, stderr, is_valid = run_command_os(f'node {mdx_parse} -f {self.file_path}', cwd=self.content_path)
+            if is_valid:
+                print_error(f'Failed verifying README.md, Path: {self.file_path}. Error Message is: {stderr}')
+                return False
+        return True
+
+    def are_modules_installed_for_verify(self) -> bool:
+        """ Check the following:
+            1. npm packages installed - see packs var for specific pack details.
+            2. node interperter exists.
+
+        Returns:
+            bool: True If all req ok else False
+        """
+        valid = True
+        # Check node exist
+        stdout, stderr, exit_code = run_command_os('node -v', cwd=self.content_path)
+        if exit_code:
+            print_warning(f'There is no node installed on the machine, Test Skipped, error - {stderr}, {stdout}')
+            valid = False
+        else:
+            # Check npm modules exsits
+            missing_packs = False
+            packs = ['@mdx-js/mdx', 'fs-extra', 'commander']
+            for pack in packs:
+                stdout, stderr, exit_code = run_command_os(f'npm ls {pack}', cwd=self.content_path)
+                if exit_code:
+                    missing_packs = True
+                    print_warning(f"The npm module: {pack} is not installed.")
+
+            # Install node modules
+            if missing_packs:
+                stdout, stderr, exit_code = run_command_os(f'npm install .', cwd=self.content_path)
```

Comment:
I don't feel comfortable doing this install in such a way. In general we shouldn't be installing stuff without asking the user. I think it is sufficient just to print a log line suggesting to run `npm install` if they want to run validation on README files.